src/content/self-hosted/manage/airgap.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: AirGap Networks
+description: Installing Okteto in AirGap Networks
+sidebar_label: AirGap Networks
+id: airgap
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+import TiersList from "@theme/TiersList";
+
+# AirGap Networks<TiersList tiers="Enterprise Self-Hosted" />
+
+Okteto works on AirGap Networks, but it requires custom configuration.
+This guide has been crafted to help you overcome these hurdles, but we encourage you to [talk to us](https://www.okteto.com/get-demo/) to guide you during the installation.
```

Comment:
```suggestion
This guide has been crafted to help you overcome these hurdles, but we encourage you to [talk to us](https://support.okteto.com/hc/en-us) to guide you during the installation.
```

Reply:
We should also rephrase this, I don't think this kind of language inspires confidence. We shouldn't say that "it's hard", "it takes more effort", "there are hurdles".  I think we should simply mention that "for airgapped environments, we recommend that you reach out to our team to discuss your requirements and scenarios" or something like that. 

For the link, since the customer is not asking for a demo but for help/guidance, I think that https://www.okteto.com/contact/ is more appropriate.